rtengine/camconst.json

```diff
@@ -1456,6 +1456,16 @@ Camera constants:
         ]
     },
 
+    { // Quality C
+        "make_model": [ "FUJIFILM GFX100 II" ],
+        "dcraw_matrix" : [ 12806, -5779, -1110, -3546, 11507, 2318, -177, 996, 5715 ], // adobe dcp d65
+        "raw_crop": [
+            // multi-aspect crop to account for 16-shot pixel shift images
+            { "frame" : [11808, 8754], "crop" : [ 0, 2, 11664, 8734 ] },
```

Comment:
The crop can be expanded to `[ 0, 0, 11664, 8750 ]` without problem. For reference, the ADC converted dng has
```
Raw Image Full Size             : 11808x8754
Raw Image Crop Top Left         : 7 8
Raw Image Cropped Size          : 11648x8736
```

The height can indeed be greater.

Reply:
Done.

The doubt that I tried to explain in the description of this PR came from the fact that the crop generated by ADC is different from the camconst.json and darktable and the crop values generated by ADC for both GFX100 and GFX100II are the same... 

FYI Rawspeed has a crop of 0, 0, -146, -6 which leads to [0, 0, 11662, 8748]

Reply:
I don't know why the raw crop for the GFX 100 is like that. Maybe it needs a review.